### PR TITLE
CLN: Removed unused code

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -406,12 +406,6 @@
         "statsmodels": ("http://www.statsmodels.org/devel/", None),
     }
 
-# extlinks alias
-extlinks = {
-    "issue": ("https://github.com/pandas-dev/pandas/issues/%s", "GH"),
```
Comment (jreback) [inline doc/source/conf.py]: this is used! its how the links in the docs show up (wiki may *not* be used though)

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/conf.py (hunk 1 above)
Comment (MomIsBestFriend) [reply doc/source/conf.py]: Oh, I tried to build the docs locally and I succeeded doing so, can you point me to where it used?
I just want to remove the use of ```%``` strings formmating.

Comment (jreback) [reply doc/source/conf.py]: when u do :issue: it replaces with this string

Comment (MomIsBestFriend) [reply doc/source/conf.py]: Oh wow, this is **__LITERALLY__** everywhere.
I really don't want to risk it.

@jreback do you think I should close this PR?

Comment (jbrockmendel) [reply doc/source/conf.py]: yes, though in general removing unused code is encouraged.  i like to use https://codecov.io/gh/pandas-dev/pandas/tree/master/pandas to find unused code (unfortunately it is out of date, hopefully thatll be fixed by #30070)

Comment (MomIsBestFriend) [reply doc/source/conf.py]: Thank you so much for the tip @jbrockmendel

**Pull request closed without merging**